src/engraving/libmscore/volta.cpp

```diff
@@ -65,8 +65,8 @@ static const ElementStyle voltaStyle {
 //   VoltaSegment
 //---------------------------------------------------------
 
-VoltaSegment::VoltaSegment(Spanner* sp, Score* s)
-    : TextLineBaseSegment(ElementType::VOLTA_SEGMENT, sp, s, ElementFlag::MOVABLE | ElementFlag::ON_STAFF | ElementFlag::SYSTEM)
+VoltaSegment::VoltaSegment(Spanner* sp, EngravingItem* parent)
```

Comment:
maybe it makes sense to require only one of these arguments?

like (Spanner* parent)

or (EngravingItem* parent) + explicit local check that parent is "Spanner"

Reply:
You right, I will fix it

Reply:
I will fix it in a next PR